src/services/photos/usage/photosUsage.service.ts

```diff
@@ -6,6 +6,11 @@ export class PhotosUsageService {
     const { usage } = await this.sdk.photos.photos.getUsage();
     return usage;
   }
+
+  // Get total count of photos
```

Comment:
Remove this comment and change the naming if is not clear, otherwise use JSDoc syntax